recipe/conda_build_config.yaml

```
@@ -4,11 +4,13 @@ MACOSX_SDK_VERSION:        # [osx and x86_64]
   - '10.14'                # [osx and x86_64]
 MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
   - '10.14'                # [osx and x86_64]
-c_compiler_version:        # [osx and x86_64]
-  - 14                     # [osx and x86_64]
-cxx_compiler_version:      # [osx and x86_64]
-  - 14                     # [osx and x86_64]
-c_compiler_version:        # [linux]
-  - 11                     # [linux]
-cxx_compiler_version:      # [linux]
-  - 11                     # [linux]
+
+# 4.23.4 is the latest protobuf migration
+# 3.21 is the version that tensorflow 2.13 was last built with
+libprotobuf:
+- '4.24.3'
+- '3.21'
+
+libgrpc:
+- '1.58'
+- '1.51'
```

Last version tensorflow was [built](https://github.com/conda-forge/tensorflow-feedstock/blob/main/.ci_support/linux_64_c_compiler_version10cuda_compilernvcccuda_compiler_version11.2cxx_compiler_version10numpy1.22python3.8.____cpython.yaml#L34) with is 1.54

You should also zip these two keys; the zip in the global pinning will eventually go away again (at least that's my goal; it was mostly to get over the 3.x -> 4.x bump)

```suggestion
- '1.54'

zip_keys:
  - 
    - libgrpc
    - libprotobuf
```